db/data/20140823134218_set_basepath_where_nil.rb

```diff
@@ -0,0 +1,14 @@
+class SetBasepathWhereNil < ActiveRecord::Migration
+  def self.up
+    OntologyVersion.where(basepath: nil).find_each do |ov|
```

Comment:
You could optimize this query by auto-loading the ontology (`includes`), and possibly by pre-selecting the fields.